app/grants/migrations/0006_auto_20190111_2350.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.1.4 on 2019-01-11 23:50
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('grants', '0005_auto_20190109_2042'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='grant',
+            name='cancel_tx_confirmed',
```

Comment:
same question here. won't this migration make it such that each existing Grant in the DB has `cancel_tx_confirmed ` == `False` ?